### PR TITLE
Add swagger support for SOFARPC restful

### PR DESCRIPTION
**Pull request by khotyn**
Add swagger support for SOFARPC restful services.

### CODE REVIEW

**Comment by khotyn** [comment]
Users can add swagger dependencies and add `com.alipay.sofa.rpc.rest-swagger=true` in applications.properties to enable swagger openapi support, the swagger openapi will be available at `http://localhost:8341/swagger/openapi`

File: sofa-boot-starter/src/main/java/com/alipay/sofa/rpc/boot/SofaBootRpcAutoConfiguration.java [hunk 1]
```diff
@@ -17,13 +17,7 @@
...
 
 import com.alipay.sofa.healthcheck.startup.ReadinessCheckCallback;
-import com.alipay.sofa.rpc.boot.config.FaultToleranceConfigurator;
-import com.alipay.sofa.rpc.boot.config.LocalFileConfigurator;
-import com.alipay.sofa.rpc.boot.config.MeshConfigurator;
-import com.alipay.sofa.rpc.boot.config.RegistryConfigureProcessor;
-import com.alipay.sofa.rpc.boot.config.SofaBootRpcConfigConstants;
-import com.alipay.sofa.rpc.boot.config.SofaBootRpcProperties;
-import com.alipay.sofa.rpc.boot.config.ZookeeperConfigurator;
+import com.alipay.sofa.rpc.boot.config.*;
```
[the remark this answers is not in the document]
Comment (khotyn) [reply sofa-boot-starter/src/main/java/com/alipay/sofa/rpc/boot/SofaBootRpcAutoConfiguration.java]: Changed as you suggested.

**Comment by khotyn** [comment]
@leizhiyuan You mean `sofaboot-dependencies`'s version?

**Comment by leizhiyuan** [comment]
ok, I see,sofaboot-dependencies 310 has released four days ago.

**Comment by leizhiyuan** [comment]
after ci，I will merge this pr to master

**Pull request merged**